### ISSUE TITLE
Feature Request: Integrate Eartrumpet functionality

### ISSUE THREAD

**Issue by Samuel12321** [ShankarBUS/ModernFlyouts#164]
One of the most requested changes from store feedback is to add individual app volume control to ModernFlyouts.

**Comment by ShankarBUS** [comment]
Woah! @Samuel12321, are you kidding me?

You do know it's very hard to do, don't you?

It will require intense work!

**Comment by Samuel12321** [comment]
hence very low priority and post v1.0 labels.

A lot of the work has already been done by the ear trumpet team. But it is something to consider in the future when the app is out of preview.

**Comment by Cyberdroid1** [comment]
Is this really required? I mean, it's just a flyout, right?

**Comment by ShankarBUS** [comment]
If it was just a flyout, I wouldn't have added new features to the GSMTC module.

LoL

@WindowsFanBoi, does this proposal somehow overlap with yours in #103?

**Comment by Cyberdroid1** [comment]
@ShankarBUS Aye Aye, captain!

Looks like it is overlapping.

**Comment by ShankarBUS** [comment]
If the EarTrumpet devs look at this and #103, they'll kick our asses lol 😂.

Our plan should not be to replace EarTrumpet. We shouldn't disgrace their app this way. Let's give it the respect it deserves 😶.

@Samuel12321, Could you edit the title to say "Ability to control volume per-app"?

And in the detail we can say similar to EarTrumpet.

But we are gonna steal code from them anyways (if we were to implement this feature) 😂.

**Comment by Cyberdroid1** [comment]
Eartrumpet is anyways open source. No trademark there. Who will stop us?

**Comment by alexeidrekker** [comment]
Perhaps a collaboration to integrate ear trumpet into modern flyout and vice versa? Not a developer, so apologies if that isn't possible and I am out of scope.

**Comment by ShankarBUS** [comment]
EarTrumpet is stable, so they won't want to have any new breaking change. Even I asked them if I could add the media controls to their app as I have some experience on that.

https://github.com/File-New-Project/EarTrumpet/issues/475

But they rejected my offer sedly 😢.

So, the only way is us integrating (stealing) features from EarTrumpet.

**Comment by Samuel12321** [comment]
technically not stealing as Ear trumpet uses a MIT licence so we can use any of their code. 
Ear trumpet seems to be not actively developed at the moment, last update was in February.

**Comment by Cyberdroid1** [comment]
Yeah, it seems to have falled to a lull moment, for now. We can "borrow" code freely. 😁😁

**Comment by alexeidrekker** [comment]
Ha ha. Fair enough. Good luck!

**Comment by riverar** [comment]
We're more than happy to replace our flyout, but we have a lot of constraints (e.g. needs to be wpf, acrylic, have all the right accessibility, controls, high dpi, animations, etc.). Does ModernFlyouts support that?

**Comment by Samuel12321** [comment]
Hi @riverar , ModernFlyouts is wpf, high dpi, has animations, and we are working on acrylic (@ShankarBUS has managed to find a way to add acrylic without the dragging lag #68). 

We are open to adding/ accepting the addition of anything you felt nessesary.

@riverar we would be more than happy to add you as a collaborator if you wanted.



@ShankarBUS any thoughts/ anything you want to add.

**Comment by ShankarBUS** [comment]
I'm just amazed! I'm too happy that they agreed to collaborate with us ❤️.

But downlevel support could be cut up to Windows 10 1809+ due to MSIX packaging.

Accessibility support needs further investigation. We need provide full accessibility support at least to the tray flyout.

Well, my acrylic method would restrict the minimum supported OS to Windows 10 1903 (due to Visual Layer dependency and reg free activation requirement).

Most of our existing user base are on Windows 10 1903+. So, I don't think this will block our path.

**Comment by Samuel12321** [comment]
@FireCubeStudios came up with an interesting [concept ](https://medium.com/@TheCubeOnFire/fluent-media-flyout-for-winglows-3411865978b5)for this, which could be helpful for future flyout designs. For now the plan is to use this in a new tray flyout.
![image](https://user-images.githubusercontent.com/35312698/137400951-b4f767eb-0afb-430c-b5a8-20119a17de12.png)

**Comment by rajeshbhavikatti** [comment]
Checkout [Volumey](https://github.com/G-Stas/Volumey) it even has an option for custom hotkeys for each application well leaving that aside looking forward for the update love to have this volume flyout view https://github.com/ModernFlyouts-Community/ModernFlyouts/issues/164#issuecomment-943767651
